bin/license/license_setup.sh

```
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set +eu
+
+source nextflow-bin/license/helper_funcs.sh
+SENTIEON_INSTALL_DIR=$(sentieon_install_dir)
+sentieon_procs=$(nproc)
```

Shellcheck (0.10.0)

[warning] sentieon_procs appears unused. Verify use (or export if used externally).

(SC2034)